fastlane_bot/events/async_event_update_utils.py

```diff
@@ -155,19 +147,9 @@ async def main_get_tokens_and_fee(c: List[Dict[str, Any]]) -> pd.DataFrame:
     )
 
 
-def pair_name(
-        t0_symbol: str,
-        tkn0_address: str,
-        t1_symbol: str,
-        tkn1_address: str,
-        key_digits: int = 4,
-) -> str:
-    return f"{t0_symbol}-{tkn0_address[-key_digits:]}/{t1_symbol}-{tkn1_address[-key_digits:]}"
-
-
-def get_pool_info(
-        pool: pd.Series,
+def _get_pool_info(
         mgr: Any,
+        pool: pd.Series,
         current_block: int,
         tkn0: Dict[str, Any],
```

Comment:
why is tkn a dict?

Reply:
This is a question for another PR.
It probably appears in other places as well.
The only reason that you're seeing a change in this function, is because I moved `mgr` to be the first input argument, in order to keep a consistent coding-convention throughout this module (in fact, it might actually imply that all functions here should actually be implemented as part of the `Manager` class rather than as a set of global functions).